### ISSUE TITLE
Long Text With No Whitespace Causes Blank Newlines

### ISSUE THREAD

**Issue by dsanders11** [electron/typescript-definitions#231]
The examples I've noticed include a long URL, but replacing it with gibberish repros the issue, so the bug must be triggered by long text with no whitespace.

```js
    /**
     * Unregisters the app from notifications received from APNS. See:
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     *
     * tion/appkit/nsapplication/1428747-unregisterforremotenotifications?language=objc
     *
     * @platform darwin
     */
    unregisterForAPNSNotifications(): void;
```
